core/roboconf-agent/src/test/java/net/roboconf/agent/internal/AgentMessageProcessorBasicTest.java

```diff
@@ -230,7 +230,12 @@ public void testSetscopedInstance_withAppBindings_andExternalExports() {
 		applicationBindings.put( "Tpl1", new HashSet<>( Arrays.asList( "app1", "app3" )));
 		applicationBindings.put( "Tpl2", new HashSet<>( Arrays.asList( "app2" )));
 
-		Message msg = new MsgCmdSetScopedInstance( app.getMySqlVm(), externalExports, applicationBindings );
+		Map<String,byte[]> scriptResources = new HashMap<> ();
+		String scriptShell = "#!/bin/bash\necho Bonjour le monde cruel > toto.txt";
+		byte[] script1 = scriptShell.getBytes();
+		scriptResources.put("script", script1);
+
+		Message msg = new MsgCmdSetScopedInstance( app.getMySqlVm(), externalExports, applicationBindings, scriptResources );
```

Comment:
This test should use null resources, as it is the basic use case.
And you should have another one where you set resources. How do you prevent regressions otherwise?